debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/AbstractStreamingAdapter.java

```diff
@@ -13,6 +13,13 @@
  * @author Chris Cranford
  */
 public abstract class AbstractStreamingAdapter implements StreamingAdapter {
+
+    protected final OracleConnectorConfig connectorConfig;
+
+    public AbstractStreamingAdapter(OracleConnectorConfig connectorConfig) {
```

Comment:
Yes, I think that's much better now than the config() approach before. Also passing the connection to the one method needing it. We should keep that, even when deciding to go back to reflective instantiation for some reason; in that case we'd just have to look up the constructor with the right signature.